backend/svirt.pm

```diff
@@ -49,10 +53,12 @@ sub do_start_vm {
         });
 
     $ssh->backend($self);
-    $self->select_console({testapi_console => 'svirt'});
 
     # remove backend.crashed
     $self->unlink_crash_file;
+
+    bmwqemu::save_vars();    # update variables
+
```

Comment:
You are separating by a lot of blank lines, also in the above paragraph. IMHO doesn't make it more readable